### ISSUE TITLE
Where to find raster data?

### ISSUE THREAD

**Issue by kmader** [swissgeodata4r/swissrastermaps4r#1]
Great package and sorry for the overly simple question, but where can I download the raster data (or is it even possible?) for the `init_fdir` and `get_raster` functions

**Comment by ratnanil** [comment]
Yeah, I realise that this information is not very clear in the documentation. If your have access to [geodata4edu](https://geodata4edu.ch), that is a great place to get the geodata for free.

Otherwiese, [Swisstopo sells their rasterdata via their homepage](https://shop.swisstopo.admin.ch/en/products/maps/national/digital/), but they also have the 1:500'000 and 1:100'000 maps for free: Check out these links:

- https://shop.swisstopo.admin.ch/en/products/maps/national/digital/srm500
- https://shop.swisstopo.admin.ch/en/products/maps/national/digital/srm1000

**Comment by ratnanil** [comment]
Updated the docs to make this clearer (6dc36a3d83c72443d2773523e9593d6b69cdea6c)
